Formula/gmt.rb

```diff
@@ -1,11 +1,11 @@
 class Gmt < Formula
   desc "Tools for manipulating and plotting geographic and Cartesian data"
   homepage "https://www.generic-mapping-tools.org/"
-  url "https://github.com/GenericMappingTools/gmt/releases/download/6.0.0/gmt-6.0.0-src.tar.xz"
-  mirror "https://mirrors.ustc.edu.cn/gmt/gmt-6.0.0-src.tar.xz"
-  mirror "https://fossies.org/linux/misc/GMT/gmt-6.0.0-src.tar.xz"
-  sha256 "8b91af18775a90968cdf369b659c289ded5b6cb2719c8c58294499ba2799b650"
-  revision 7
+  url "https://github.com/GenericMappingTools/gmt/releases/download/6.1.0/gmt-6.1.0-src.tar.xz"
+  mirror "ftp://ftp.soest.hawaii.edu/gmt/gmt-6.1.0-src.tar.xz"
+  mirror "https://mirrors.ustc.edu.cn/gmt/gmt-6.1.0-src.tar.xz"
+  mirror "https://fossies.org/linux/misc/GMT/gmt-6.1.0-src.tar.xz"
```

Comment:
How do we need three mirrors instead of two?

Reply:
Two mirrors are enough. The first one is the GMT official FTP server, so I just removed the third mirror.

Reply:
👍